Aptoide/bds-sdk
# feature/appc 1784 store/oem attribution on pay as guest

**What does this PR do?**

- This PR adds a new way to find the OEM id that is inserted in a apk. For that we added a external library.

**Database changed?**
 No

**Where should the reviewer start?**

- AdyenPaymentFragment.java
- OemIdExtractorService.java
- OemIdExtractorV2.java
- OemIdExtractorV1.java

**How should this be manually tested?**

Better talk to me about how we should test this since its not a simple flow.

Flow on how to test this or QA Tickets related to this use-case: [APPC-1784](https://aptoide.atlassian.net/browse/APPC-1784)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1784](https://aptoide.atlassian.net/browse/APPC-1784)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 

Yes, extractor.jar


**Code Review Checklist**
